### PR TITLE
board/rtl8721csm: Change key_id used for WEP shared key auth method to 0

### PR DESCRIPTION
**Pull request by fengyunlai**
Change key_id from 1 to 0 for WEP shared key auth method connection according to RTK default SDK setting, also to fix connection issue found in compatibility test.

### CODE REVIEW

File: os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c [hunk 1]
```diff
@@ -445,7 +445,7 @@ int8_t cmd_wifi_connect(trwifi_ap_config_s *ap_connect_config, void *arg)
 		password = ap_connect_config->passphrase;
 		ssid_len = strlen((const char *)ssid);
 		password_len = ap_connect_config->passphrase_length;
-		key_id = 1; //Foucus
+		key_id = 0;
```
Comment (sangam-swami) [inline os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c:448]: you can initialize with zero at declaration, as nowhere it is getting modified or assigned different values

Comment (jeongchanKim) [inline os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c:448]: I agreed Sangam's opinion. And in this function(`cmd_wifi_connect()`), all `key_id` are used as 0, so it is better to initialize at the first of the function.

**Review by jeongchanKim — APPROVED** [review]
LGTM

File: os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c (hunk 1 above)
Comment (fengyunlai) [reply os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c]: I agree with you, thanks

Comment (fengyunlai) [reply os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c]: I agree with you, thanks

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**